pyramid_swagger/api.py

```diff
@@ -74,3 +74,25 @@ def view_for_api_declaration(request):
             basePath=str(request.application_url),
         )
     return view_for_api_declaration
+
+
+def register_swagger_json_endpoint(config):
+    """Registers the endpoint /swagger.json.
+
+    :param config: Configurator instance for our webapp
+    :type config: :class:`pyramid.config.Configurator`
+    """
+    spec = config.registry.settings['pyramid_swagger.spec']
+
+    def view_for_swagger_json(request):
+        # Thanks to the magic of closures, this means we gracefully return JSON
+        # without file IO at request time.
+        return spec.spec_dict
+
+    route_name = 'swagger_json'
+    config.add_route(route_name, '/swagger.json')
+    config.add_view(
+        view_for_swagger_json,
+        route_name=route_name,
+        renderer='json'
+    )
```

Comment:
This pull request is for 2.0 support, not 1.2 and 2.0 co-existing.  If #95 is useful during the 1.2 + 2.0 co-existing effort, then that is wonderful.  I've separated these concerns for many reasons.  It makes things simpler, allows iteration, and produces results.